### PR TITLE
Fixing acceptance tests with values out of range.

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@jframos 

### DESCRIPTION
- Fixing acceptance tests with values out of range.

### CODE REVIEW

File: tests/acceptance/features/component/context_update.feature [hunk 1]
```diff
@@ -116,15 +116,15 @@ Feature: Receive context update requests
...
-      | qatest    | 1.05  | 0.8     | 0.1   | 0.15    |
-      | qatest    | 0.05  | 5.8     | 0.1   | 0.15    |
-      | qatest    | 0.05  | 0.8     | 10.1  | 0.15    |
-      | qatest    | 0.05  | 0.8     | 0.1   | 415.15  |
-      | qatest    | -0.05 | 0.8     | 0.1   | 0.15    |
-      | qatest    | 0.05  | -0.8    | 0.1   | 0.15    |
-      | qatest    | 0.05  | 0.8     | -0.1  | 0.15    |
-      | qatest    | 0.05  | 0.8     | 0.1   | -0.15   |
+      | server_id | cpu    | memory  | disk  | network |
+      | qatest    | 100.05 | 0.8     | 0.1   | 0.15    |
```
Comment (flopezag) [inline tests/acceptance/features/component/context_update.feature]: range is between 0 and 100 or between 0 and 1?

Comment (geonexus) [inline tests/acceptance/features/component/context_update.feature]: 0-100

**Comment by jframos** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Comment by flopezag** [comment]
LGTM

**Pull request merged**